tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/sampleNeutrino/1/NeutrinoInput.nescene

```diff
@@ -27,9 +27,9 @@
             <property name="Verbose" hidden="0" locked="0" val="false"/>
             <property name="Threads" hidden="0" locked="0" val="6"/>
             <property name="ServerPort" hidden="0" locked="0" val="20200"/>
-            <property name="SceneFilePath" hidden="0" locked="0" val="C:\msys64\home\RYANED\Emerald\projects\raven\tests\framework\CodeInterfaceTests\Neutrino\Neutrino_Base\sampleNeutrino\1\NeutrinoInput.nescene"/>
-            <property name="SaveDir" hidden="0" locked="0" val="C:\msys64\home\RYANED\Emerald\projects\raven\tests\framework\CodeInterfaceTests\Neutrino\Neutrino_Base\sampleNeutrino\1\"/>
-            <property name="CacheDir" hidden="0" locked="0" val="C:\msys64\home\RYANED\Emerald\projects\raven\tests\framework\CodeInterfaceTests\Neutrino\Neutrino_Base\sampleNeutrino\1\"/>
+            <property name="SceneFilePath" hidden="0" locked="0" val="/Users/talbpw/raven/tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/sampleNeutrino/1/NeutrinoInput.nescene"/>
+            <property name="SaveDir" hidden="0" locked="0" val="/Users/talbpw/raven/tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/sampleNeutrino/1/"/>
+            <property name="CacheDir" hidden="0" locked="0" val="/Users/talbpw/raven/tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/sampleNeutrino/1/"/>
```

Comment:
why the unique absolute path is used here? very confusing for me.

Reply:
Same for all NeurinoInput

Reply:
fixed